app/src/main/java/com/github/onlynotesswent/model/cache/FolderDao.kt

```diff
@@ -0,0 +1,27 @@
+package com.github.onlynotesswent.model.cache
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.github.onlynotesswent.model.folder.Folder
+
+@Dao
+interface FolderDao {
+  @Query("SELECT * FROM folder WHERE id = :folderId") fun getFolderById(folderId: String): Folder?
+
+  @Query("SELECT * FROM folder") fun getFolders(): List<Folder>
```

Comment:
I named it like this since the uid will always be the one of the current user, as you stated below. I can change it for consistency though and to allow potential changes in the future